providers/zoho.go

```diff
@@ -53,5 +60,38 @@ func init() {
 				LogoURL: "https://res.cloudinary.com/dycvts6vp/image/upload/v1722471890/media/zoho_1722471890.svg",
 			},
 		},
+		DefaultModule: ZohoCRM,
+		Modules: &Modules{
+			ZohoCRM: {
+				BaseURL:     "https://www.zohoapis.com",
+				DisplayName: "Zoho CRM",
+				Support: Support{
+					Read:      true,
+					Subscribe: false,
+					Write:     true,
+				},
+			},
+			ZohoDeskV2: {
+				BaseURL:     "https://desk.zoho.com",
```

Comment:
Guessing this has similar multi-datacenter rules as the other stuff, please check